From 8a1aa0318ce13ed241b7d75950b333d9c4edbe16 Mon Sep 17 00:00:00 2001
From: ivan-borodiei <ivan.borodiei@gmail.com>
Date: Sun, 5 Jan 2020 18:10:40 +0200
Subject: [PATCH] Edit ex message

---
 Market/Market.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Market/Market.cs b/Market/Market.cs
index 38573f3..07b589b 100644
--- a/Market/Market.cs
+++ b/Market/Market.cs
@@ -118,7 +118,7 @@ namespace ConsoleApp
 
             if (price == null)
             {
-                throw new Exception("Product price has not been found");
+                throw new Exception("Product price has not been found!");
             }
 
             productList.Add(price.Product);
-- 
2.24.1.windows.2

